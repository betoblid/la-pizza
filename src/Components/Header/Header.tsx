
import Logo from "../../assets/logo.png"

export const Header = () => {

    return(
        <header className="bg-banner bg-center bg-no-repeat bg-cover py-20">
            <img src={Logo} alt="Logo pizzaria" className="sm:w-96 w-1/2 mx-auto" />
        </header>
    )
}