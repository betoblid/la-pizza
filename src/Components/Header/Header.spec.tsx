import { render, screen } from "@testing-library/react"
import { Header } from "./Header"


describe("<Header />", () => {

    it("verificar se o Header estÃ¡ na tela", () => {
        render(<Header />)

        expect(screen.getByAltText(/pizzaria/i)).toBeInTheDocument()
    })
    it("criar um snapshot", () => {
        const {container } = render(<Header />)

        expect(container).toMatchSnapshot()
    })
})