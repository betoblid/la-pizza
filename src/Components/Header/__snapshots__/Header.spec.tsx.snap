// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> criar um snapshot 1`] = `
<div>
  <header
    class="bg-banner bg-center bg-no-repeat bg-cover py-20"
  >
    <img
      alt="Logo pizzaria"
      class="sm:w-96 w-1/2 mx-auto"
      src="logo.png"
    />
  </header>
</div>
`;
